docs/recipes/create-immortal-spore.md

```diff
@@ -1,15 +1,24 @@
 # Create immortal spores on-chain
 
-Immortal is a Spore Extension that provides the following features:
-- An immortal spore will live on-chain forever and cannot be destroyed 
-- Spore Extensions can only be added on a spore on the spore's creation
+## What is immortal
```

Comment:
What is `immortal` / What is Immortal Attribute (?)

// just putting "immortal" there without anything else looks weird